plugins/module_utils/switch_take_ownership_client.py

```diff
@@ -47,7 +47,7 @@ def __get_host_address(self):
                     return (host_ip_address, host_ip_mask, host_mac_address)
         raise Exception(f'could not find a host ip address in the same network as the switch {self._switch_ip_address}')
 
-    def __get_default_host_address(self):
+    def __get_default_interface(self):
```

Comment:
Can you please rename this method to `__get_default_host_interface`? It better reflects its intent.